trackpy/linking.py

```diff
@@ -474,6 +476,12 @@ def link_df(features, search_range, memory=0,
         the next frame.
 
         For examples of how this works, see the "predict" module.
+    adaptive_step : float (optional)
```

Comment:
This is picky, but I have been parsing numpydoc formatted strings recently and the convention is 

```
name : type, optional
```